orographic_precipitation/tests/fixture_orographic_precipitation.py

```diff
@@ -8,7 +11,7 @@ def input_params():
 
     params = {
         'latitude': 0,
-        'precip_base': 0,
+        'precip_base': np.ones((140,140)),
```

Comment:
Ideally we should test both the spatially uniform (scalar) and variable (array) cases, but probably fine to leave this like this for now.